addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -207,6 +234,14 @@ export const many2ManyTagsField = {
             type: "string",
             help: _t("Write a domain to allow the creation of records conditionnally."),
         },
+        {
+            label: _t("Edit Tags"),
+            name: "edit_tags",
+            type: "boolean",
+            help: _t(
+                "If checked, users will be able to edit tag related record via clicking on tag."
```

Comment:
"If checked, users will be able to edit tag related records by clicking on tags."